pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_podinfoes.yaml

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
delete this file. this file is no longer getting auto-generated right?

Reply:
I am not sure, maybe codegen is generating it ?

Reply:
try deleting both of these files and run codegen again to see which files get generated

Reply:
sure let me do that.

Reply:
podinfoes.yaml is still being generated with make codegen. how do I fix that ?

Reply:
struggle

Reply:
make generate is generating that file.

Reply:
looks like adding this line fixes the issue https://github.com/cilium/tetragon/pull/1369/files#diff-bc54ba3950ae7bb8c718f46e20d8c315f8ff7661d6950fe362276e8160b26227R277